src/lib/execution_plan_test.rs

```diff
@@ -315,6 +315,54 @@ cd -"#
     assert!(task.env.is_none());
 }
 
+#[test]
+#[cfg(target_os = "linux")]
+fn create_workspace_task_with_included_members() {
+    let mut crate_info = CrateInfo::new();
+    let members = vec![
+        "member1".to_string(),
+        "member2".to_string(),
+        "dir1/member3".to_string(),
+    ];
+    crate_info.workspace = Some(Workspace {
+        members: Some(members),
+        exclude: None,
+    });
+
+    envmnt::remove("CARGO_MAKE_USE_WORKSPACE_PROFILE");
```

Comment:
we are not testing profile switch, so we can remove this one and just put 'development' hardcoded in the line